src/librbd/internal.cc

```diff
@@ -3587,6 +3691,11 @@ namespace librbd {
 	bl.append(buf + q->first, q->second);
       }
 
+      r = ictx->update_object_map(p->objectno, OBJECT_EXISTS);
```

Comment:
Hmm ... I don't think the object map update was even in the AioRequest state machine.  The biggest change from the initial implementation is the ability to do mass object map updates instead of one call per object.